Aztec/Classes/Converters/HTMLNodeToNSAttributedString.swift

```diff
@@ -249,17 +249,20 @@ private extension HTMLNodeToNSAttributedString {
             return attributes
         }
 
-        let representation = HTMLRepresentation(for: .element(HTMLElementRepresentation(element)))
+        let elementRepresentation = HTMLElementRepresentation(element)
+        let representation = HTMLRepresentation(for: .element(elementRepresentation))
         var finalAttributes = attributes
 
+        let ignoredAttributes = [StandardElementType.li.rawValue]
```

Comment:
Can we roll this change back please?

This should be tackled and discussed as part of #578, and doesn't really belong to the scope of this PR.